sale_margin_delivered/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Sale Margin Delivered",
-    "version": "12.0.1.0.1",
+    "version": "12.0.1.0.2",
```

Comment:
No need to do this, as ocabot can do the version bumping upon merge

Reply:
As there's a migration script, it's force to ensure the right path